rotkehlchen/tests/unit/decoders/test_makerdao.py

```diff
@@ -61,3 +62,135 @@ def test_makerdao_simple_transaction(
             extra_data={'vault_type': 'ETH-A'},
         ),
     ]
+
+
+@pytest.mark.vcr()
+@pytest.mark.parametrize('ethereum_accounts', [['0xa217BDa86b0EDb86eE7d4D6e34F493eDF1ea4F29']])  # noqa: E501
+def test_makerdao_sdai_redeem_transaction(
+        database,
+        ethereum_inquirer,
+):
+    """Data taken from
+    https://etherscan.io/tx/0x6b2a1f836cfc7c28002e4ac60297daa6d79fcde892d9c3b9ca723dea2f21af5c
+    """
```

Comment:
Not needed. We know the transaction hash from the contents of the test. Only add docstring here if you need to explain anything

```suggestion
```